app/controllers/person/sac_remarks_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+class Person::SacRemarksController < ApplicationController
+  before_action :set_remark
+  before_action :authorize_action, only: %i[edit update]
+
+  def index
+    authorize!(:show_remarks, entry)
+  end
+
+  def update
+    return render :edit unless @person.update(params.require(:person).permit(Person::SAC_REMARKS))
+
+    respond_to do |format|
+      format.html { render '_remark', layout: false, locals: { remark: @remark } }
+    end
+  end
+
+  private
+
+  def entry
+    @group ||= Group.find(params[:group_id])
+    @person ||= Person.find(params[:person_id])
+  end
+
+  def set_remark
```

Comment:
wir versuchen in ruby set nicht zu verwenden.
einfach ein getter machen? vom naming her wäre evtl. `sac_remark_attr` sprechender, was meinst du?